src/ocean/THCM.H

```diff
@@ -344,22 +341,6 @@ public:
     //! set if you have vmix_flag=1 in mix_imp.f (recommended).
     void fixMixing(int value);
 
-    //! \name get physical global domain bounds
-    //@{
-    inline double xMin() const {return xmin;}
-    inline double xMax() const {return xmax;}
-    inline double yMin() const {return ymin;}
-    inline double yMax() const {return ymax;}
-    inline double hDim() const {return hdim;}
-    //@}
-
-    //! \name these are used by the block preconditioner:
-    //!@
-    inline double AlphaT()    const {return alphaT;}
```

Comment:
I think I want to have the possibility to extract alphaT and alphaS, but these were probably not functioning anyway right?